src/activity/factory.ts

```diff
@@ -13,6 +17,7 @@ import { GolemInternalError } from "../error/golem-error";
  */
 export class ActivityFactory {
```

Comment:
I'm just wondering... how will the user be able to listen to the `activityCreated` event? `ActivityFactory` is not public... it is just a helper for the `Activity.create` function.. so in future low level api, it would be:
```ts
agreement.createActivity();
agreement.events.on('activityCreated');
// underneath, if agreement uses a factory to create activity, it will have to pass this event from the factory to its events

activityPool.events.on('activityCreated'); // or current workService
```
but for current `TaskExecutor` If I want to somehow catch and pass all events from golem-js under the `golemEvents` field, how can I do this?